docs/hazmat/primitives/asymmetric/serialization.rst

```diff
@@ -489,6 +489,41 @@ file suffix.
         ``additional_certificates`` is a list of all other
         :class:`~cryptography.x509.Certificate` instances in the PKCS12 object.
 
+.. function:: serialize_key_and_certificates(password, name, key, cert, cas,
+                                             key_encryption, cert_encryption)
+
+    .. versionadded:: 3.0
+
+    Serialize a PKCS12 blob.
+
+    :param name: The friendly name to use for the supplied certificate and key.
+    :type name: :term:`bytes-like`
+
+    :param password: The password to use to encrypt the data. ``None``
+        if the PKCS12 should not be encrypted.
+    :type password: :term:`bytes-like`
```

Comment:
This doesn't match the other in the signature.

Also the encryption algorithms have a password, so why does this have an additional password argument?

Reply:
They should not, but also you can only have one password with OpenSSL's API so we'll need to enforce that both passwords are the same (if present).

Reply:
Err, it should not have this argument. Not sure why I wrote "They should not"

Reply:
Or we can only allow passing one parameter for the time being :-)